distributed/security.py

```diff
@@ -123,7 +126,8 @@ def get_connection_args(self, role):
         """
         d = {}
         tls = self.get_tls_config_for_role(role)
-        d['ssl_context'] = self._get_tls_context(tls, ssl.Purpose.SERVER_AUTH)
+        purpose = ssl.Purpose.SERVER_AUTH if hasattr(ssl, "Purpose") else None
```

Comment:
I'd like to see a comment added here to mention compatibility with old Python 2s.